### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by Maverick-08**
added footer markup and style

### CODE REVIEW

**Review by VinayOjha1 — CHANGES_REQUESTED** [review]
Change footer height

**Review by Maverick-08 — COMMENTED** [review]
what should be the height

**Comment by VinayOjha1** [comment]
> what should be the height

75

File: style.css [hunk 1]
```diff
@@ -1,12 +1,31 @@
...
+    height: 70px;
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: teal;
+}
+
+.footer{
+    height: 70px;
```
Comment (VinayOjha1) [inline style.css:25]: change here

**Review by VinayOjha1 — APPROVED** [review]

**Pull request merged**